x-pack/platform/plugins/shared/cases/server/client/alerts/get.ts

```diff
@@ -37,3 +42,41 @@ export const getAlerts = async (
     ...alert._source,
   }));
 };
+
+export const getAlertMetadataFromComments = async (
+  comments: AttachmentAttributes[],
+  clientArgs: CasesClientArgs
+): Promise<CaseMetadata> => {
+  const alertInfo: AlertInfo[] = comments.flatMap((c) => {
```

Comment:
Why not just use: https://github.com/elastic/kibana/blob/5146f827b3c794d7e77ad89fb87e1f492f7a1bf0/x-pack/platform/plugins/shared/cases/server/common/utils.ts#L199 ?

Reply:
I didn't see that, thanks!

Done [here](https://github.com/elastic/kibana/pull/230192/commits/ae83d75eaa71cc4c88788f64357635a1082ff439).